plugins/package-managers/spdx/src/main/kotlin/utils/SpdxResolvedDocument.kt

```diff
@@ -133,6 +133,16 @@ internal data class SpdxResolvedDocument(
         return pkg
     }
 
+    /**
+     * Retrieve the issues from [issuesByReferenceId] that are not associated with [any package][packagesById]. These
+     * issues can be related to general issues within the SPDX document.
+     */
+    fun getIssuesWithoutSpdxPackage(): List<Issue> {
+        return issuesByReferenceId.mapNotNull { (id, issue) ->
```

Comment:
Ah, sorry, forgot to mention this earlier: This should become an expression function as well.